src/cloud/env/variables-global.md

```diff
@@ -50,11 +50,11 @@ stage:
 The `SCD_ON_DEMAND` variable skips the SCD in both phases (build and deploy), clears the `pub/static` and `var/view_preprocessed` folders, and writes the following to the `app/etc/env.php` file:
 
 ```php?start_inline=1
-return array(
+return [
```

Comment:
When SCD_ON_DEMAND is set to true, the `env.php` reflects the following change in the remote Cloud environment:
<img width="348" alt="Screen Shot 2020-07-31 at 09 42 03" src="https://user-images.githubusercontent.com/5817319/89046354-755b2100-d312-11ea-95a6-d7f74af77ae7.png">

Please revert changes to the Cloud topics.

Reply:
I will, but didnt understand why use the old array syntax in example.

Reply:
In this example, we show the outcome of adding the environment variable to the configuration file. After deploying the new configuration, this is what Cloud automatically adds to the `env.php` file. We don't have control over that and we are not advocating manual changes to the file in this topic.

Reply:
It's strange. I'll check later whats happening in the cloud. 
I have removed the change for now.

Reply:
@hguthrie kindly approve.